src/data/source/emissions.yml

```diff
@@ -0,0 +1,11 @@
+carbon_intensity_factors:
+  2014: 137
```

Comment:
Can you add a data source? If we're adding a new source we need to maintain year-over-year, we'll need to have that noted, and make sure that it errors out if the latest year's intensity is missing